services/brig/test/integration/API/User/Client.hs

```diff
@@ -962,6 +965,29 @@ testNewNonce brig = do
         Just "no-store" @=? getHeader "Cache-Control" response
       pure nonceBs
 
+-- todo(leif): work in progress
+testCreateAccessToken :: Brig -> Http ()
+testCreateAccessToken brig = do
+  uid <- userId <$> randomUser brig
+  cid <- randomClient
+  let proof = Proof $ "xxxx." <> Base64.encode claims <> ".zzzz"
+  response <- Util.createAccessToken brig uid cid proof <!! const 204 === statusCode
+  liftIO $
+    assertBool "DPoP header should contain a byte string" $ isJust (getHeader "DPoP" response)
+  where
+    claims =
+      ""
+        <> "{"
+        <> "    \"jti\": \"7535d380-673e-4219-8410-b8df679c306e\","
+        <> "    \"iat\": 1653455836315,"
+        <> "    \"htm\": \"POST\","
+        <> "    \"htu\": \"https://wire.example.com/client/token\","
+        <> "    \"nonce\": \"OZ-5WL9vR5CxaaXoMWpcZQ\","
+        <> "    \"chal\": \"okAJ33Ym/XS2qmmhhh7aWSbBlYy4Ttm1EysqW8I/9ng\","
+        <> "    \"sub\": \"URI:wireapp:SvPfLlwBQi-6oddVRrkqpw/04c7@example.com\","
+        <> "    \"exp\": 1661231836315"
+        <> "}    "
```

Comment:
this is outdated, right?  we can probably get one from the test suite of the rust library that we're calling here?